src/contract.ts

```diff
@@ -0,0 +1,381 @@
+import {
+  Keypair,
+  PublicKey,
+  TransactionInstruction,
+  SYSVAR_CLOCK_PUBKEY,
+} from '@solana/web3.js';
+import { ethers } from 'ethers';
```

Comment:
Ethers is kind of sizeable but we're only using a small part of it here. It would be great to import the packages needed to enable tree shaking and avoid duplication.

Reply:
gotcha. done.